src/app.ts

```diff
@@ -41,8 +41,11 @@ export namespace testApp {
 
     getApp(): firebase.app.App {
       if (typeof this.appSingleton === 'undefined') {
+        const config = process.env.FIREBASE_CONFIG
```

Comment:
Do we know what happens when `firebase.initializeApp()` is initialized w/ an empty config?

Things won't work as intended in some strange way. Blowing up `JSON.parse(undefined)` ins't a great experience either, but curious what problem we are fixing here.

Reply:
Yeah this one was interesting.

Tossing an empty config didn't seem to do much of anything. I'm down for any other alternative.